dna/pDna/pDna.go

```diff
@@ -0,0 +1,43 @@
+package pDna
+
+import "math"
+
+// Float32Base encodes a DNA base as a probability vector using float32 precision.
+// Note that gap probabilities can be stored implicitly as 1 - (A + C + G + T).
+type Float32Base struct {
```

Comment:
I missed the discussion at code review, but I thought the idea was to make base probabilities float16's in memory, not on disk. Did you guys decide to make the bases float32's in memory instead?

Reply:
Sorry, it's float16 on disk, float32 in memory. The idea is we'll never try to store a whole genome in memory at once, usually reading will be paired with something like fasta.Extract, where we pull our subsequence of interest out of a large file. That's the vision, obviously missing the context in that the Extract command for pFa is still pending